src/views/Tournament/Tournament.tsx

```diff
@@ -1152,7 +1149,7 @@ export class Tournament extends React.PureComponent<TournamentProperties, any> {
                     }
                     {!editing && !loading &&
                         <div>
-                            {(((data.get("user").is_tournament_moderator || data.get("user").id === tournament.director.id)
+                            {(((data.get("user").is_tournament_moderator || data.get("user").id === tournament.director.id || true)
```

Comment:
Hmm, this `|| true` makes the edit button visible to everyone, I'm assuming that wasn't the intent - was there another intent here, or was that just a debug thing that we can take out?

Reply:
Yes, that true shouldn't be there. Removed